en/12_Project_Governance/09_Repository_Management.md

```diff
@@ -0,0 +1,71 @@
+---
+title: GitHub repository management
+summary: Information about management of the various GitHub repositories
+iconBrand: github
+---
+
+# GitHub repository management
+
+This document outlines the rules and processes for managing the GitHub repositories for supported modules.
+
+## Branches
+
+### Naming convention
+
+Branches with a single integer (e.g. `5`) represent either the next *major* release (if there have been no tagged releases for that release line yet) or the next *minor* release.
+
+Branches with an integer, a dot, and another integer (e.g. `5.2`) represent either the *next patch release* for the current minor, or a patch release for a previous minor version.
+
+> [!NOTE]
+> Branches with any other naming strategy should not be the target of any pull requests. Only branches following this naming convention are included in merge-ups and are used as the base for new tagged releases.
```

Comment:
This prevents chaining PRs (if we say we can't target other branches than `X` or `X.Y`), do we care about that? We don't use it much, but it's not inconceivable it may become handy at some point.

Reply:
> This prevents chaining PRs (if we say we can't target other branches than X or X.Y)

I'm not sure what you mean? We generally don't even have branches that don't following that documented naming convention... can you please give me an example so I can wrap my head around it?

Reply:
So say you'd wanted to replace React with Svelte for CMS 6 as a longer running initiative, you could create `6-svelte` from where a PR would target `6` and partial PRs would target `6-svelte`. But there are other ways how to go around that in terms of branching and PRs and it could be a separate repo etc., so probably rather an edge case where I got triggered by the wording that some branches shouldn't be a PR target.

Reply:
Yeah I think we'd handle that in another way - or we can always update this documentation if that is a branching strategy we choose to adopt. Right now that's not a process we have so I think trying to handle it here is beyond the scope of this PR.